examples/Demos/SofaDefaultScene.scn

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+
+<Node name="root" dt="0.01" gravity="0 0 9.81" >
+    <!-- Loading the dynamic libraries required for this scene -->
+    <RequiredPlugin name="PluginManager"
+        pluginName="Sofa.Component.IO.Mesh Sofa.Component.LinearSolver.Direct Sofa.Component.LinearSolver.Iterative
+        Sofa.Component.Mapping.Linear Sofa.Component.Mass Sofa.Component.ODESolver.Backward Sofa.Component.Setting
+        Sofa.Component.SolidMechanics.FEM.Elastic Sofa.Component.StateContainer Sofa.Component.Topology.Container.Dynamic
+        Sofa.Component.Visual Sofa.GL.Component.Rendering3D Sofa.Component.AnimationLoop Sofa.Component.Collision.Detection.Algorithm
+        Sofa.Component.Collision.Detection.Intersection Sofa.Component.Collision.Geometry Sofa.Component.Collision.Response.Contact
+        Sofa.Component.Constraint.Lagrangian.Solver Sofa.Component.Constraint.Lagrangian.Correction Sofa.Component.LinearSystem
+        Sofa.Component.MechanicalLoad MultiThreading Sofa.Component.SolidMechanics.Spring Sofa.Component.Constraint.Lagrangian.Model
+        Sofa.Component.Mapping.NonLinear Sofa.Component.Topology.Container.Constant Sofa.Component.Topology.Mapping BeamAdapter Sofa.GUI.Component
+        Sofa.Component.Topology.Container.Grid Sofa.Component.Engine.Select" />
+
+    <!-- Rendering settings -->
+    <VisualStyle name="RenderingOptions" displayFlags="showVisualModels" />
+    <BackgroundSetting color="0.8 0.8 0.8 1" />
+
+    <!-- Define Mouse left click as a Bilateral Lagrangian constraint -->
+    <ConstraintAttachButtonSetting /> <!-- The presence of this component sets the mouse interaction to Lagrangian-based constraints at the GUI launch -->
```

Comment:
This has not been tested on imgui. I guess this is why it does not work. A work has been started to support all kind of interactions: https://github.com/sofa-framework/SofaGLFW/pull/129